tests/test_main.py

```diff
@@ -18,15 +19,25 @@
 out = os.path.join(cwd, 'dist')
 
 
+_DEFAULT_BACKEND = {
```

Comment:
Not really important, but it might be a good idea to make this a fixture or the result of a function call, so that you can make a fresh copy each time, since dictionaries are mutable (not that you mutate it, but it would be annoying to track down a bug if you accidentally started mutating it).

Reply:
Yes. I did not do it because we only use in 1 place, so I thought it was better to not over optimize. If we need it in another place we can make it a fixture :)

Reply:
I would argue that you should probably put it in-line in the single test that you're using rather than at module level if you are relying on the fact that it's only used in one place. You can always refactor it out into a fixture or function later if you need the same thing, and "future you" (or me, or some random contributor) is not going to know the history of this default value, and during review the implications of "we are re-using a mutable data structure" may be missed.

Reply:
That is fair :)

Reply:
Well, actually I could put it in the package.